packages/backend/src/persistance/storage/getUserStorage.ts

```diff
@@ -3,9 +3,14 @@ import { UserStorage } from 'dm3-lib-storage';
 import { getIdEnsName } from '../getIdEnsName';
 
 export function getUserStorage(redis: Redis) {
```

Comment:
I wonder if we could use a more concise naming for that method since it is used to retrive a certain storage slot key. 
Maybe something like getStorageSlot / readSlot